public/app/features/connections/pages/EditDataSourcePage.tsx

```diff
@@ -1,24 +1,13 @@
 import * as React from 'react';
 import { useLocation, useParams } from 'react-router-dom';
 
-import { Page } from 'app/core/components/Page/Page';
-import { EditDataSource } from 'app/features/datasources/components/EditDataSource';
-import { EditDataSourceActions } from 'app/features/datasources/components/EditDataSourceActions';
-
-import { useDataSourceSettingsNav } from '../hooks/useDataSourceSettingsNav';
+import DataSourceTabPage from 'app/features/datasources/components/DataSourceTabPage';
 
 export function EditDataSourcePage() {
   const { uid } = useParams<{ uid: string }>();
   const location = useLocation();
   const params = new URLSearchParams(location.search);
   const pageId = params.get('page');
-  const { navId, pageNav } = useDataSourceSettingsNav();
 
-  return (
-    <Page navId={navId} pageNav={pageNav} actions={<EditDataSourceActions uid={uid} />}>
-      <Page.Contents>
-        <EditDataSource uid={uid} pageId={pageId} />
-      </Page.Contents>
-    </Page>
-  );
+  return <DataSourceTabPage uid={uid} pageId={pageId} navId="connections-connections-datasources" />;
```

Comment:
```suggestion
  return <DataSourceTabPage uid={uid} pageId={pageId} navId="connections-datasources" />;
```

thin this navId has changed